src/gordon_gcp/plugins/janitor/authority.py

```diff
@@ -202,18 +210,27 @@ def _create_msgs(self, instances):
     async def run(self):
         """Batch instance data and send it to the :obj:`self.rrset_channel`.
         """
+        timer = self.metrics.timer('janitor-authority-runtime')
+        timer.start()
```

Comment:
IIRC this is a coroutine, so it should be `await timer.start()` (and same as `await timer.stop()`)